src/config/nfs.md

```diff
@@ -0,0 +1,43 @@
+# NFS
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` package:
+
+```
+# xbps-install nfs-utils
+```
+
+Before you mount an NFS share, you'll need to [enable the statd and procbind
+services](services/index.md#enabling-services):
+
+(If your server supports `nfs4`, you do not need statd and you can replace `nfs`
+with `nfs4` below)
+
+```
+# sudo ln -s /etc/sv/rpcbind /var/service
+# sudo ln -s /etc/sv/statd /var/service
+```
+
+Now you're ready to mount a directory:
+
+```
+# mount -t nfs <host:/path/to/dir> /mnt/nfs/share
+```
+
+For example, if you wanted to connect `/volume` on a server at IP Address
+`192.168.1.99` to a directory `/mnt/volume` on your local system, you could do:
+
+```
+# mkdir /mnt/volume
+# mount -t nfs 192.168.1.99:/volume /mnt/volume
+```
+
+To have the directory mounted again when the system reboots, add an entry to
+`/etc/fstab`:
```

Comment:
```
`/etc/fstab` -> [fstab(5)](https://man.voidlinux.org/fstab.5)
```

Also, find some way to mention https://man.voidlinux.org/nfs.5 and https://man.voidlinux.org/nfs.conf.5 (this last one should probably go near the top instead).

Reply:
I added links to each, but don't know anything about them so please copywrite